### PR TITLE
Lab 1

### PR DESCRIPTION
**Pull request by cyraalesha**

### CODE REVIEW

File: Cyra/part_1/src/test.cpp
Comment (RCMast3r) [inline Cyra/part_1/src/test.cpp:1]: please dont include empty files that arent being used.

File: Cyra/part_1/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(hello_world LANGUAGES CXX)
+add_library(one STATIC src/test.cpp include/random.h)
```
Comment (RCMast3r) [inline Cyra/part_1/CMakeLists.txt:4]: in general it is preferred to instead use target_include_directories for setting your include directory that contains your header files.

also why do you have a header here that is empty? there isnt a need for this in this step

File: Cyra/part_1/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+project(part_1)
```
Comment (RCMast3r) [inline Cyra/part_1/CMakeLists.txt:5]: why do you have a second project call here? there should only ever be one project() call in a single cmake project

File: Cyra/part_1/flake.nix
Comment (RCMast3r) [inline Cyra/part_1/flake.nix:1]: i know i did this, and included a flake.nix in my folder, however this isnt needed yet.

File: Cyra/part_1/include/random.h
Comment (RCMast3r) [inline Cyra/part_1/include/random.h:1]: see comment about empty files being included

**Review by RCMast3r — CHANGES_REQUESTED** [review]
please put this part 1 folder submission into a folder called lab1 or something like it.

(I know i didnt do this but I should have)

**Comment by cyraalesha** [comment]
Just committed the revision. My bad, I forgot to delete the random stuff I was trying

**Pull request closed without merging**